src/main/webapp/app/exam/participate/events/exam-live-events-overlay.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="overlay-wrapper">
+    <div class="events-wrapper">
+        <ng-container *ngFor="let event of eventsToDisplay">
+            <jhi-exam-live-event [event]="event" [showAcknowledge]="unacknowledgedEvents.includes(event)" (onAcknowledge)="acknowledgeEvent(event)" />
+        </ng-container>
+    </div>
+    <button *ngIf="unacknowledgedEvents.length === 0" class="btn btn-primary w-100 mt-2" (click)="closeOverlay()">Close</button>
```

Comment:
Wrong button. I mean the notification button to open the list of events.

Reply:
My initial idea was to not give the student another element of distraction in an exam where the feature is not used at all. What do you think about this? I think your POV is also good though

Reply:
I agree, the typical exam should not need these announcements. Also, a specific working time change is quite more likely than a change of the complete exam, so only the affected student should see the notification button.

Reply:
Fair point. In this case, there is not really a right or wrong.

It stood out for me because it took about 3 seconds to appear in the video, which is weird.